### PR TITLE
[PL-48456] SMP 0.14.4 patch release notes

### PR DESCRIPTION
**Pull request by kapilgarg1996**
SMP 0.14.4 patch release notes

For reviewers, a preview is available:

[Self-Managed Enterprise Edition](https://65fb464e7788091e2aa131ea--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -15,15 +15,17 @@ import delete_project from './static/delete-project.png'
 
 These release notes describe recent changes to Harness Self-Managed Enterprise Edition, NextGen.
 
-:::danger important upgrade instructions for patch release 0.13.4 and 0.12.1
+:::danger important upgrade instructions for patch release 0.14.4, 0.13.4 and 0.12.1
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:18]:
```suggestion
:::danger important upgrade instructions for patch releases 0.14.4, 0.13.4, and 0.12.1
```

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — DISMISSED** [review]

File: release-notes/self-managed-enterprise-edition.md [hunk 2]
```diff
@@ -15,15 +15,17 @@ import delete_project from './static/delete-project.png'
...
 
-:::danger important upgrade instructions for patch release 0.13.4 and 0.12.1
+:::danger important upgrade instructions for patch releases 0.14.4, 0.13.4, and 0.12.1
 
 If you are currently on version 0.12.0, you must follow the applicable upgrade process below to upgrade your version to the latest stable release, 0.12.1.
 
 If you are currently on version 0.13.0, 0.13.1, 0.13.2, or 0.13.3, you must follow the applicable upgrade process below to upgrade your version to the latest stable release, 0.13.4.
 
-You can perform your normal upgrade process if you are currently on a version earlier than 0.12.0. Harness recommends that you upgrade to 0.13.4.
+If you are currently on version 0.14.3, you must follow the applicable upgrade process below to latest stable release, 0.14.4.
```
Comment (pratmit) [inline release-notes/self-managed-enterprise-edition.md:24]: What is the applicable upgrade process? Where are we publishing that?

File: release-notes/self-managed-enterprise-edition.md [hunk 3]
```diff
@@ -15,15 +15,17 @@ import delete_project from './static/delete-project.png'
...
 
-**Upgrade using Helm**
+You can perform your normal upgrade process if you are currently on a version earlier than 0.12.0. Harness recommends that you upgrade to 0.14.4.
+
+**Upgrade using Helm to version 0.12.1 OR 0.13.4**
```
Comment (brian-f-harness) [inline release-notes/self-managed-enterprise-edition.md:28]:
```suggestion
**Upgrade using Helm to version 0.12.1, 0.13.4, or 0.14.14**
```

File: release-notes/self-managed-enterprise-edition.md (hunk 2 above)
Comment (brian-f-harness) [reply release-notes/self-managed-enterprise-edition.md:24]: @pratmit I added a comment to add 0.14.4 to the heading below on line 28. @kapilgarg1996 is this correct, or are there additional steps required for 0.14.4 upgrade?

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**